numpy/core/_internal.py

```diff
@@ -631,4 +631,17 @@ class TooHardError(RuntimeError):
     pass
 
 class AxisError(ValueError, IndexError):
-    pass
+    """ Axis supplied was invalid. """
```

Comment:
right now, `numeric.py` has `from ._internal import TooHardError, AxisError`

`core/__init__.py` also contains `from . import _internal  # for freeze programs`, for some reason